swcc/swcc/models.py

```diff
@@ -409,8 +406,13 @@ def load_data_spreadsheet(self, file: Union[Path, str]) -> Dataset:
             elif data_type == Mesh:
                 subject.add_mesh(file=shape_file, anatomy_type='unknown')
 
-            # TODO: where to find the modality?
-            subject.add_image(file=image_file, modality='unknown')
+            # The image_file column is optional
+            if row[1]:
```

Comment:
Before, we were testing that column[1] had the explicit name of `image_file`.  Now, this just assumes it is column 1 but doesn't test.  Perhaps this should be `if 'image_file' in headers and row[headers.index('image_file')]:` so as to not assume the column position.

Reply:
Good point, I will fix that. that's what the `expected = ('shape_file', 'image_file')` above does.